src/components/analyses/listing/TableView.js

```diff
@@ -62,12 +62,22 @@ function AnalysisName(props) {
     const NameLink = () => {
         return (
             <Tooltip
+                id={build(baseId, ids.ANALYSIS_NAME_CELL, ids.TOOLTIP)}
+                aria-label={formatMessage(intl, "goOutputFolderOf", {
+                    name: name,
+                })}
                 title={formatMessage(intl, "goOutputFolderOf", { name: name })}
             >
                 <Link
+                    id={build(baseId, ids.ANALYSIS_NAME_CELL)}
                     onClick={() => handleGoToOutputFolder(analysis)}
                     color="primary"
                     component="button"
+                    aria-describedby={build(
+                        baseId,
+                        ids.ANALYSIS_NAME_CELL,
+                        ids.TOOLTIP
+                    )}
```

Comment:
Ah, my comment wasn't meant to instruct you to add an `aria-describedby` for Tooltips.

What I meant was that MUI does it automatically for us, with their `Tooltip` component.
So if MUI's `Tooltip` is adding an `aria-describedby` automatically to this `Link`, then it makes sense to me that the `Link` doesn't need an extra `aria-label`, especially if we follow MUI's examples and add the `aria-label` to the `Tooltip` (as you have done above).

In other words, we can leave the `aria-label` on the `Tooltip`, but we don't need `aria-describedby` for children of Tooltips, since MUI adds them automatically.